data/campaigns/World_Conquest/lua/game_mechanics/wocopedia/help_dialog.lua

```diff
@@ -93,13 +94,13 @@ return {
 							horizontal_grow = true,
 							vertical_grow = true,
 							T.tree_view {
-								id = "left_tree",
+								id = "treeview_topics",
```

Comment:
This isn't directly related to the new API, but I think it might be much cleaner if you put the dialog definigion into an actual WML file and loaded it in with `wml.load`... unless it's dynamically-built in a way that would prevent this?

Reply:
dunno, i like lua so personally wont take the time to do that.. , i don't think its dynamicially  created though.